.github/workflows/tests.yml

```
@@ -12,7 +12,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        variants: [ {typo3: 11.5, php: 7.4}, {typo3: 12.4, php: 8.1} ]
+        variants: [ {typo3: 11.5, php: 8.1}, {typo3: 12.4, php: 8.1}, {typo3: 12.4, php: 8.3}  ]
```

Why not run tests for all supported combinations?
```suggestion
        variants: [ {typo3: 11.5, php: 8.1}, {typo3: 11.5, php: 8.2}, {typo3: 11.5, php: 8.3}, {typo3: 12.4, php: 8.1}, {typo3: 12.4, php: 8.2}, {typo3: 12.4, php: 8.3} ]
```

---

> Why not run tests for all supported combinations?

Yes, this should be supported for manual CI runs.

I don't expect that it's worth running all combinations each time. Each CI run costs resources. Covering the corner cases (oldest and newest supported releases of TYPO3 and PHP) is hopefully sufficient to detect all possible problems.